### ISSUE TITLE
module entry in package.json wrong

### ISSUE THREAD

**Issue by jonbjk** [davidkpiano/useEffectReducer#13]
The module entry in package.json is:
https://github.com/davidkpiano/useEffectReducer/blob/master/package.json#L56
```
  "module": "dist/useEffectReducer.esm.js",
```

However the built file is:
```
dist/use-effect-reducer.esm.js
```
thereby breaking usage in rollup

**Comment by davidkpiano** [comment]
Can you make a PR to fix this?

**Comment by jonbjk** [comment]
https://github.com/davidkpiano/useEffectReducer/pull/14

**Comment by davidkpiano** [comment]
Closed by #14, thanks!

**Comment by jonbjk** [comment]
@davidkpiano Did I do something wrong in the PR? The version number did not update when you merged the PR. I thought it would update to 0.6.1

**Comment by jonbjk** [comment]
... in other words. Could you please push a fix build to npm, so that I can use useEffectReducer in my (rollup based) project.

**Comment by davidkpiano** [comment]
Published as 0.6.1
